src/libmongoc/tests/test-mongoc-background-monitoring.c

```diff
@@ -24,6 +24,7 @@
 #include "mongoc/mongoc-client-pool-private.h"
 #include "test-libmongoc.h"
 #include "TestSuite.h"
+#include "test-conveniences.h"
```

Comment:
Probably makes no difference, but is this typically ordered before `test-libmongoc.h`?